frontend/src/components/input/editor/MentionList.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+	<div class="mention-items">
+		<template v-if="items.length">
+			<button
+				v-for="(item, index) in items"
+				:key="item.id"
+				class="mention-item"
+				:class="{ 'is-selected': index === selectedIndex }"
+				@click="selectItem(index)"
+			>
+				<img
+					:src="item.avatarUrl"
+					alt=""
+					class="mention-avatar"
+				>
+				<div class="mention-info">
+					<p class="mention-name">
+						{{ item.label }}
```

Comment:
If the user did not set a name, the label will be the same as the username, maybe we can just hide it in that case?

Reply:
<img width="352" height="272" alt="image" src="https://github.com/user-attachments/assets/94200c8b-6c5e-4d70-a687-5784bd5d05ff" />

Do you mean this?